src/PlaywightSharp.ProtocolTypesGenerator/PlaywrightSharp.ProtocolTypesGenerator.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <StyleCopTreatErrorsAsWarnings>false</StyleCopTreatErrorsAsWarnings>
+    <CodeAnalysisRuleSet>../PlaywrightSharp.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Text.Json" Version="4.7.1" />
+  </ItemGroup>
+
+  <Import Project="../Common/Dependencies.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\PlaywrightSharp.Abstractions\PlaywrightSharp.Abstractions.csproj" />

```

Comment:
Let's add our analyzers here as well

Reply:
they are included through `<Import Project="../Common/Dependencies.props" />`

Reply:
We are missing these guys there right https://github.com/kblok/playwright-sharp/blob/master/src/PlaywrightSharp.Chromium/PlaywrightSharp.Chromium.csproj#L9-L11 ?

Reply:
yeah. just one of them though